Source/WebKit/PlatformWPE.cmake

```diff
@@ -120,6 +120,8 @@ list(APPEND WebKit_SERIALIZATION_IN_FILES
     Shared/glib/UserMessage.serialization.in
 
     Shared/soup/WebCoreArgumentCodersSoup.serialization.in
+
+    Shared/wpe/AvailableInputTypes.serialization.in
```

Comment:
This patch is for now wpe specific, but we could use the same approach for gtk, so I would add this in a glib dir instead of wpe. Also `InputTypes` can be confusing name, since we use the same for the form inputs, see `InputType.h`. I would use AvailableInputDevices instead, or something similar.